recipes/nupic/LICENSE.txt

```diff
@@ -0,0 +1,661 @@
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
Please delete this license file. Conda will get the one from the downloaded source from pypi.